devices/SparkfunLCD/SparkfunLCD.cs

```diff
@@ -0,0 +1,763 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.SparkfunLCD
+{
+    using System;
+    using System.Device.I2c;
+    using System.Threading;
+    using nanoFramework.Hardware.Esp32;
```

Comment:
This should not be part of the binding but part of the example as you don't set the pins in the binding so it stays generic.